test/fluentd-forward.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+# This is a test suite for the fluentd secure_forward feature
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/deployer/scripts/util.sh"
```

Comment:
Once all of the tests are ported from hack/testing to test, we should move or get rid of deployer/